float64 AxCalib
float64 AyCalib
float64 AzCalib
float64 Ax
float64 Ay
float64 Az
float64 AxRaw
float64 AyRaw
float64 AzRaw
float64 Vx
float64 Vy
float64 Vz
float64 Gx
float64 Gy
float64 Gz
float64 currTime
